### PR TITLE
Update G8 Template

### PR DESCRIPTION
**Pull request by pomadchin**
This PR fixes packages names, updates plugin deps, updates sbt, scalafix rules upd.

### CODE REVIEW

File: src/main/g8/application/src/main/scala/database/util/GeotrellisWktMeta.scala [hunk 1]
```diff
@@ -1,76 +0,0 @@
-package $package$.database.util
```
Comment (pomadchin) [inline src/main/g8/application/src/main/scala/database/util/GeotrellisWktMeta.scala:1]: Anyway it is a very outdated code.

File: src/main/g8/application/src/main/scala/$package__packaged$/api/Server.scala [hunk 2]
```diff
@@ -44,7 +44,7 @@ object Server extends IOApp {
         )
       ).orNotFound
       server <- {
-        BlazeServerBuilder[IO]
+        BlazeServerBuilder[IO](connectionEc)
```
Comment (jisantuc) [inline src/main/g8/application/src/main/scala/$package__packaged$/api/Server.scala:47]: This is surprising to me -- why should the server builder use the same context as database interaction?

**Review by jisantuc — COMMENTED** [review]
Largely :+1: -- could you also bump the scala version in `build.sbt`? I think with the other dependency upgrades it should be free https://github.com/pomadchin/azavea.g8/blob/fix/packages/src/main/g8/build.sbt#L68

File: src/main/g8/application/src/main/scala/$package__packaged$/api/Server.scala (hunk 2 above)
Comment (pomadchin) [reply src/main/g8/application/src/main/scala/$package__packaged$/api/Server.scala]: @jisantuc lol I used whatever first to make compiler happy.

**Pull request merged**